storage/src/main/java/zipkin2/storage/kafka/KafkaStorage.java

```diff
@@ -349,32 +292,17 @@ void ensureTopics() {
 
   void doClose() {
     try {
-      if (adminClient != null) adminClient.close(1, TimeUnit.SECONDS);
+      if (adminClient != null) adminClient.close(Duration.ofSeconds(1));
       if (producer != null) {
         producer.flush();
-        producer.close(1, TimeUnit.SECONDS);
+        producer.close(Duration.ofSeconds(1));
       }
       if (traceStoreStream != null) {
         traceStoreStream.close(Duration.ofSeconds(1));
       }
-      if (traceRetentionStream != null) {
```

Comment:
judging solely by the less things to close, seems a simplicity win :)

Reply:
:raised_hands: